services/systemd/swaync.service.in

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Swaync notification daemon
+Documentation=https://github.com/ErikReider/SwayNotificationCenter
+PartOf=graphical-session.target
+After=graphical-session.target
+Requisite=graphical-session.target
+# ConditionEnvironment requires systemd v247 to work correctly
+ConditionEnvironment=WAYLAND_DISPLAY
+
+[Service]
+Type=dbus
+BusName=org.freedesktop.Notifications
+ExecStart=@bindir@/swaync
+ExecReload=@bindir@/swaync-client --reload-config --reload-css
```

Comment:
The client can atm only handle one argument so `--reload-css` does not get fired

Reply:
Oh, didn't notice.
THne, removing the entire ExecReload should handle it better, as for what i know, it simply stops and starts the service again.

Reply:
> Oh, didn't notice. THne, removing the entire ExecReload should handle it better, as for what i know, it simply stops and starts the service again.

This may be a hacky solution but why not run `@bindir@/swaync-client --reload-config; @bindir@/swaync-client --reload-css`? 🙃

Reply:
Well, not exactly hacky per se, but would systemd like the fact that it reloads in two steps?

Reply:
Ok, systemd doesn't complain, but to be sure I used && instead of ; , so that if the first operation fails for whatever reason, it doesn't carry out the second one.

Reply:
Cool! Other than these two issues, the code LTGM! :)

Reply:
Done, see https://github.com/ErikReider/SwayNotificationCenter/pull/50/commits/d481a9bb950e7786433eeebc63228a9d55a5854b and thanks :)